data/scrape/lib/generate_summary.py

```diff
@@ -114,10 +112,69 @@ def main() -> None:
                 f"| {count:,} |",
                 file=sink,
             )
+    # adding KPI code
+    # Currently hard-coded; what's a good alternative?
+    TSV_PATH = "../tsv_summary.tsv"
+    table = pandas.read_csv(
+        TSV_PATH,
+        sep="\t",
+        names=[
+            "link",
+            "iso_639_2_code",
+            "iso_639_language_name",
+            "wiktionary_language_name",
+            "script",
+            "dialect",
+            "filtered",
+            "narrow_broad",
+            "case_folding",
+            "no_entries",
+        ],
+    )
+    # Somewhat arbitrarily, we use the ISO 639-2 code as the "real" label for
+    # languages.
+    languages = table.iso_639_2_code.nunique()
+    print(f"Languages:\t{languages:,}")
```

Comment:
Isn't this printing this to the command line too? We don't need that. Ditto with all the other print statements not to the Markdown or TSV file below.